FinniversKit/Sources/Base Components/ViewModelCoordinator/AnyViewModelCoordinator.swift

```diff
@@ -0,0 +1,31 @@
+// MARK: - ViewModelCoordinator + Extension
+
+extension ViewModelCoordinator {
+    public func eraseToAnyCoordinator() -> AnyViewModelCoordinator<Route> {
+        return .init(coordinator: self)
+    }
+}
+
+// MARK: - AnyViewModelCoordinator
+
+public final class AnyViewModelCoordinator<Route>: ViewModelCoordinator {
+
+    // MARK: - Properties
+
+    internal let _handle: (Route) -> ()
```

Comment:
The access specifier should be enough for access semantics, there is no need for underscore prefix on the name.

Reply:
hehe, yeah 😉 it's a copy-pasted code 😛